content/content-strategy/audit-content/prove-value/supporting-takeaway.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
!! This file duplicates `content/content-strategy/audit-content/prove-value/key-takeaway.md` @trisha-t it appears that _this_ is the file being referenced in the topic's YAML file. Recommend remove the `key-takeaway.md` because it will not be used.

Reply:
Trisha fixed